### DISCUSSION TITLE
sensor status

### DISCUSSION THREAD

**Discussion by chadlangston** [Security-Onion-Solutions/securityonion#3322] [Unsupported Versions]
Hello, I am running a standalone security onion server with two sensor interfaces and one management interface. It was installed using the so-network-setup on CentOS 7. I am pretty sure that I configured both of my sensors during the setup, but I can't seem to find a way to corroborate that I am doing PCAP on both sensors. Is there a way to test this?

Thank you!

**Comment by weslambert** [comment]
You can try using something like tcpreplay to replay to the individual interfaces, then search for the related events/PCAP.

**Reply by chadlangston** [reply]
Thanks Wes. Is there any way to check the SO config to see which interfaces are configured as sensors?
